launcher/Application.h

```diff
@@ -74,6 +76,13 @@ class ITheme;
 class MCEditTool;
 class ThemeManager;
 class IconTheme;
+class BaseInstance;
+
+struct MinecraftTarget;
+
+// pointers for lazy people
+using InstancePtr = std::shared_ptr<BaseInstance>;
+using MinecraftTargetPtr = std::shared_ptr<MinecraftTarget>;
```

Comment:
Definitely not a good idea - InstancePtr is also defined in other headers
Somehow it's already defined in 2 other headers... but that's not great :/